### PR TITLE
Updating readme with versioning and publishing instructions

### PR DESCRIPTION
**Pull request by hannahbergam**
The versioning and publishing process has several easy-to-forget steps, so this update will hopefully be a start in documenting those necessary steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,33 @@ Now you should be able to run all the tests:
 And spin up a development build of your new project:
 
     yarn build
+
+### Publishing a new version
+
+1. In /maze: yarn login with an authorized npm account. If necessary, create one under your own email, login with dev account and add your new account to the org. After yarn login, you may need to also authorize your machine with npm adduser (follow the propmt given).
```
Comment (jmkulwik) [inline README.md:29]: Rather than `yarn login`, `npm login` was needed.

File: README.md [hunk 2]
```diff
@@ -23,3 +23,33 @@ Now you should be able to run all the tests:
...
+
+2. In /maze: 
+
+    git checkout main
```
Comment (jmkulwik) [inline README.md:33]: Also `git pull`

File: README.md [hunk 3]
```diff
@@ -23,3 +23,33 @@ Now you should be able to run all the tests:
...
+
+3. Verify yarn build and yarn test run as expected.
+
+4. Update the version:
+
+    npm version [major|minor|patch|premajor|preminor|prepatch].
+
+5. Publish the new version:
+
+    npm publish
```
Comment (jmkulwik) [inline README.md:43]: This step ended up being skip-able if I used `npm login`

Comment (hannahbergam) [reply README.md:43]: Ahh- as in, creating the version automatically published it, yes?

File: README.md (hunk 2 above)
Comment (hannahbergam) [reply README.md:33]: 🙌

File: README.md (hunk 1 above)
Comment (hannahbergam) [reply README.md:29]: 🙌

**Review by jmkulwik — APPROVED** [review]
Thank you for adding these details!

**Pull request merged**